Application/EdFi.Ods.SandboxAdmin/appsettings.json

```diff
@@ -19,6 +19,7 @@
   "DefaultOperationalContextUri": "uri://ed-fi-api-host.org",
   "OAuthUrl": "http://localhost:54746/oauth/",
```

Comment:
should this be empty and added in the developers settings file?

Reply:
Done.